templates/two-columns/two-columns.js

```diff
@@ -0,0 +1,13 @@
+import { loadBlock } from '../../scripts/lib-franklin.js';
+
+export default function buildAutoBlocks() {
+  const columnsBlock = document.querySelector('.columns:has(.form)');
```

Comment:
the two column template shouldn't be form specific, it should layout two sections as columns and that's it. The content of the section should be handled by normal block code as they would if no special template would be applied.